### DISCUSSION TITLE
Supabase Storage Move Object API issue

### DISCUSSION THREAD

**Discussion by imknight** [supabase/supabase#3062] [Questions] [answered]
it seems like the move object API is not working and the doc is outdated 
https://supabase.github.io/storage-api/#/

anyone can share tips if manage to move object through API ?

**Comment by dshukertjr** [comment, marked as answer]
Hi @imknight!

I can confirm that move object endpoint is working and it is possible to move an object as I just tried it out and worked! Are you calling the API via Supabase client? Are you getting any errors?

**Reply by imknight** [reply]
yup, they just fixed it without saying anything

**Comment by eMeRiKa13** [comment]
I'm trying to move an image, using the js library, but I always have `{ message: 'The resource was not found', status: 400 }` even if I can find my image in the table editor and the storage database.

Move is still working well, I suppose?

**Reply by sctgraham** [reply]
I have this same error  `{ message: 'The resource was not found', status: 400 }`. 

I'm using the Supabase JS API with the `service_role` on a NextJS API route and can't seem to figure out why this is happening. I've tried adding policies for SELECT / UPDATE to the `service_role` under `storage.objects` but still no luck. 

Does anyone have an idea how to move files with the `service_role` ? Cheers.
